docs/getting-started/symfony-bundle.md

```diff
@@ -12,47 +12,52 @@ Install it using [composer package manager](https://getcomposer.org/download/) i
 
 ```bash
 composer require pheature/symfony-toggle
+composer require pheature/inmemory-toggle
+composer require pheature/php-sdk
+```
+
+```php
+<?php
+// config/bundles.php
+return [
+    ...,
+    Pheature\Community\Symfony\PheatureFlagsBundle::class => ['all' => true],
+];
+
 ```
 
 ### Config
 
 ```yaml
 # config/packages/pheature-flags.yaml
-parameters:
-  pheature.inmemory.config:
-    feature_1:
-      id: feature_1
-      enabled: false
-      strategies: [ ]
-    feature_2:
-      id: feature_2
-      enabled: true
-      strategies: [ ]
-      
-services:
-  Pheature\InMemory\Toggle\InMemoryConfig:
-    class: Pheature\InMemory\Toggle\InMemoryConfig
-    arguments:
-      - '%pheature.inmemory.config%'
-
-  Pheature\InMemory\Toggle\InMemoryFeatureFactory:
-    class: Pheature\InMemory\Toggle\InMemoryFeatureFactory
-
-  Pheature\Core\Toggle\Read\FeatureFinder:
-    class: Pheature\InMemory\Toggle\InMemoryFeatureFinder
-    arguments:
-      - '@Pheature\InMemory\Toggle\InMemoryConfig'
-      - '@Pheature\InMemory\Toggle\InMemoryFeatureFactory'
-
-  Pheature\Core\Toggle\Read\Toggle:
-    class: Pheature\Core\Toggle\Read\Toggle
-    arguments:
-      - '@Pheature\Core\Toggle\Read\FeatureFinder'
-
-  Pheature\Sdk\CommandRunner:
-    class: Pheature\Sdk\CommandRunner
-    arguments:
-      - '@Pheature\Core\Toggle\Read\Toggle'
+pheature_flags:
+  driver:               ~ # One of "inmemory"; "dbal"
+  prefix:               ''
+  segment_types:
+    - { type: 'identity_segment', factory_id: 'Pheature\Model\Toggle\SegmentFactory' }
+    - { type: 'strict_matching_segment', factory_id: 'Pheature\Model\Toggle\SegmentFactory' }
```

Comment:
Here we could also add the `in_collection_matching_segment`. What do you think?

Reply:
Is a nice point;-D, we will create a new issue to add the `in_collection_matching_segment` to the default loaded Segments in Symfony config.
![image](https://user-images.githubusercontent.com/1093654/116747443-33c0e900-a9fe-11eb-8445-2b88f39fe3f1.png)

When we make the bundle it doesn't already exist.